### PR TITLE
pkg(go-1.16.2): Added go 1.16.2

### PR DESCRIPTION
**Pull request by vfrazao-ns1**
Added Go to piston v3.

### CODE REVIEW

File: packages/go/1.16.2/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+wget https://golang.org/dl/go1.16.2.linux-amd64.tar.gz
```
Comment (HexF) [inline packages/go/1.16.2/build.sh:2]: wget isn't installed in the container - use curl instead

**Review by HexF — CHANGES_REQUESTED** [review]

File: packages/go/1.16.2/run [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+GOCACHE=$PWD go run $*
```
Comment (HexF) [inline packages/go/1.16.2/run:3]: Use `"$@"` instead of `$*`

**Review by HexF — CHANGES_REQUESTED** [review]

**Review by HexF — APPROVED** [review]

**Comment by HexF** [comment]
^^vv testing CI (this package is faster than ruby to build)

**Comment by vfrazao-ns1** [comment]
This fixes issue 116: https://github.com/engineer-man/piston/issues/116

**Review by HexF — APPROVED** [review]

**Pull request merged**